packages/github/package.json

```diff
@@ -38,8 +38,9 @@
   },
   "dependencies": {
     "@actions/http-client": "^1.0.3",
-    "@octokit/graphql": "^4.3.1",
-    "@octokit/rest": "^16.43.1"
+    "@octokit/core": "^2.5.1",
+    "@octokit/plugin-paginate-rest": "^2.2.0",
+    "@octokit/plugin-rest-endpoint-methods": "^3.10.0"
```

Comment:
Curious whether there is a big impact to overall size on disk?

Reply:
There shouldn't be, @octokit/rest imported all 3 of these modules. i'll check but it should be less